cmd/registry/user.go

```diff
@@ -44,7 +44,7 @@ var usersListCmd = &cobra.Command{
 }
 
 var userGetCmd = &cobra.Command{
-	Use:   "user",
+	Use:   "userByPath",
```

Comment:
the original name `user` is the preferred way, as it's consistent with the CRUD naming convention and used in other commands (get node, get service, get environment, etc.)

Reply:
Changed to `user`